lib/hutch/broker.rb

```diff
@@ -292,13 +292,18 @@ def parse_uri
 
       u = URI.parse(@config[:uri])
 
+      @config[:mq_tls]      = u.scheme == 'amqps'
       @config[:mq_host]     = u.host
-      @config[:mq_port]     = u.port
+      @config[:mq_port]     = u.port || default_mq_port
       @config[:mq_vhost]    = u.path.sub(/^\//, "")
       @config[:mq_username] = u.user
       @config[:mq_password] = u.password
     end
 
+    def default_mq_port
+      @config[:mq_tls] ? 5671 : 5672
```

Comment:
Please switch to `AMQ::Protocol::DEFAULT_PORT` and `AMQ::Protocol::TLS_PORT`.

Reply:
Thanks @michaelklishin , ♻️